### DISCUSSION TITLE
Custom Fields in the Report Template

### DISCUSSION THREAD

**Discussion by prasanmgc** [hrcorval/behavex#168] [Ideas]
Is it possible to add custom fields to the Report? Also I would like to change the existing texts of the report (like title, header, footer) to meet organisation standards. Multiple-Cucumber-HTML-Reporter supports that. but need similar functionality to this wrapper.
There could be a provision to explicitly pass the output_strings.py to the wrapper via some config is also great.

**Comment by hrcorval** [comment]
Hi @prasanmgc, we are working on a simple functionality that will allow adding more information to the test execution report. This functionality will be ready pretty soon (within the next 2 months)
Keep you posted on that.
